src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2621,6 +2622,47 @@ class misc_commandscript : public CommandScript
         handler->GetSession()->SendShowMailBox(player->GetGUID());
         return true;
     }
+    
+    static bool HandleNearGraveyard(ChatHandler* handler, char const* /*args*/)
+    {
+        Player* player = handler->GetSession()->GetPlayer();
+
+        float x = player->GetPositionX();
+        float y = player->GetPositionY();
+        float z = player->GetPositionZ();
+
+        const WorldSafeLocsEntry* nearestLoc = nullptr;
+        bool isNear = false;
+        float distNearest = 20000.0f;
```

Comment:
you could use std::numeric_limits<float>::max() from <limits> and simplify the code in the for loop